### ISSUE TITLE
Do not register custom json serializer implicitly instead provide registration information in the docs

### ISSUE THREAD

**Issue by emorozov** [django-money/django-money#636]
Hello,
djmoney registers its own json deserializer for django-reversion in the `models/__init__.py`. This seems to be inconvenient, because:
a) such registration will override any custom serializers set by user in the `settings.py` (`SERIALIZATION_MODULES`) causing errors that are pretty difficult to debug
b) it is hard to override djmoney override because it is done at model import time, if I need my custom serializer for json, where would I put the registration? When I put it into `settings.py` which is the standard Django approach, it's overriden by djmoney. If I put registration in my models, there's no guarantees that it would be run after djmoney registration, the code would become dependent on module loading order, hence hard to understand and flaky.

I suggest removing automatical registration and instead asking the user to explicitly register djmoney serializer in `settings.py` if needed (e.g. if django-reversion is used in the project).

**Comment by benjaoming** [comment]
I agree with this, if you can open a PR with the new behavior and put a good release note, it sounds good to me. We could target it for 2.2.

**Comment by dbartenstein** [comment]
@emorozov: thanks for raising this issue. The [Django geojson serializer](https://docs.djangoproject.com/en/3.2/ref/contrib/gis/serializers/) also gets kicked out by django-money:

**Comment by fhavret** [comment]
I created a PR to fix this issue, as it doesn't seem to be resolved.
Can I please have some feedback ? https://github.com/django-money/django-money/pull/729
